### ISSUE TITLE
[Awaiting payment a while ago] System message for approval has no Spanish translation

### ISSUE THREAD

**Issue by m-natarajan** [Expensify/App#39200]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 1.4.57-2
**Reproducible in staging?:** y
**Reproducible in production?:** y
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** @JmillsExpensify 
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1711593424889649

## Action Performed:
1. Set user a’s app preferences to Spanish
2. Submit a request as user a
3. Have user b approve the request from user a

## Expected Result: 
System message are translated in Spanish, if that is the user’s language preference

## Actual Result: 
The approval system message isn’t translated, while other system messages are

## Workaround:
unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [ ] Android: mWeb Chrome
- [ ] iOS: Native
- [ ] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

Add any screenshot/video evidence

![CleanShot 2024-03-27 at 21 29 26@2x](https://github.com/Expensify/App/assets/38435837/31264912-6436-4cb6-beaa-2f666be30cf3)

![Screen Shot 2024-03-28 at 1 22 38 PM](https://github.com/Expensify/App/assets/38435837/7b28f7fc-4fa7-4385-abf9-894d15356be6)

</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~012ebbcb5ad5a18675</li>
        <li>Upwork Job ID: 1773410815992803328</li>
        <li>Last Price Increase: 2024-03-28</li>
    </ul>
</details>

**Comment by marcochavezf** [comment]
Assigning myself, since it's related to the collect approvers implementation and requires an internal fix

**Comment by isabelastisser** [comment]
Added to wave-collect.

**Comment by marcochavezf** [comment]
I think this can be considered a polish issue, I will move it to weekly to focus on other higher-priority issues, but feel free to change if you disagree

**Comment by isabelastisser** [comment]
Not overdue.

**Comment by marcochavezf** [comment]
Working on a hold request bug and awareness campaign for track expense newsletter

**Comment by isabelastisser** [comment]
No updates.

**Comment by isabelastisser** [comment]
@marcochavezf, I will put this on hold until you're able to prioritize it. Thanks!

**Comment by marcochavezf** [comment]
No update, we will need to merge first https://github.com/Expensify/Auth/pull/10883

**Comment by isabelastisser** [comment]
Still on hold.

**Comment by marcochavezf** [comment]
PR up

**Comment by isabelastisser** [comment]
@marcochavezf, what are the next steps here? All done?

**Comment by trjExpensify** [comment]
Yep, but @dukenv0307 reviewed the PR and needs paying @isabelastisser

**Comment by isabelastisser** [comment]
@dukenv0307, I sent you an offer in Upwork now. Please accept it and I will process the payment. Thanks!

**Comment by isabelastisser** [comment]
All set so closing this.
